lua/astrocommunity/lsp/actions-preview-nvim/init.lua

```diff
@@ -0,0 +1,28 @@
+local keymaps = {
+  ["<Leader>la"] = {
+    function() require("actions-preview").code_actions() end,
+    desc = "LSP code action",
+    cond = "testDocument/codeAction",
+  },
+}
+
+local mappings = {
+  n = keymaps,
+  v = keymaps,
+}
+
+---@type LazySpec
+return {
+  "aznhe21/actions-preview.nvim",
+  event = "User AstroLspSetup",
```

Comment:
```suggestion
  lazy = true,
```

This should be much more aggressively lazy loaded. And only get loaded by module since it's fully driven by a single keybind.